member_data_history/models/hr_employee.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2019 Coop IT Easy SCRL fs
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+def format_partner(partner):
+    return "({}, {})".format(partner.id, partner.name)
```

Comment:
Why store the `_id` here too, and not with `type_id`

Reply:
Because parner name can change but the id is not human readable. So I put both. I could do that with type_id but it felt like it changes much less and that the name of the type at a time is what matters.